doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -52,49 +53,102 @@ For this guide, we'll create a getting started project in a branch of your fork
 ```text
 sdk\<service name>\<package name>\README.md
 sdk\<service name>\<package name>\api
+sdk\<service name>\<package name>\perf
+sdk\<service name>\<package name>\samples
 sdk\<service name>\<package name>\src
+sdk\<service name>\<package name>\stress
 sdk\<service name>\<package name>\tests
-sdk\<service name>\<package name>\samples
 sdk\<service name>\<package name>\CHANGELOG.md
+sdk\<service name>\<package name>\<package name>.sln
 ```
 
 - `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
+
+### Use Cadl as Input
+
+#### Create Cadl project
   
-We will use dotnet project template [Azure.Template](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template) to automatically create the project.
+  ***Initialize Cadl Project***
 
-You can run `eng\scripts\automation\Invoke-DataPlaneGenerateSDKPackage.ps1` to generate the starting SDK client library package directly as following:
+  Follow [Cadl Getting Start](https://github.com/microsoft/cadl/#using-node--npm) to initialize your Cadl project.
```

Comment:
I think here, service team should already have their CADL files ready, so can we ask them to download their cadl project directly instead of initializing a new project.

Reply:
I will add statement that they can download cadl project from swagger repo or init one.